### ISSUE TITLE
Should load Keyboard layer before login in

### ISSUE THREAD

**Issue by ghost** [KaOSx/apps#93]
#### Submission type
  - [ ] Bug report
Kaos Linux do not load specific keyboard layer (here french) before accessing to the log in window. So this behavior imply that the user type a wrong password or mus do an annoying intellectual gymnastic to like using a english keyboard. 
=> I'm French, so I've chosen a french keyboard layer.
=> But when rebooting and in front of "SDDM" logging window, my password wich have an "a" couldn't be recognize because it is a "z" on english keyboard layer.

#### Package name & version you are having issues with or have a request for
> SDDM-kcm version 1:5.8.5.1-1
> other ?

#### Used repositories
> kaos native

#### Expected behaviour
> load specific keyboard layer before the loggin window
> load the specific keyboard layer before kde because if having any problem we can log in terminal without "window manager" and use normal typing instead of annoying virtual "key" replacement.

#### Actual behaviour
> loading to late the specific keyboard (after logging and starting the KDE desktop) with the risk of  cannot be able to type the good password.

#### Steps to reproduce the problem
>running on a mac air so I choose "apple portable" with "french" fr 

#### System info link created by Octopi > Tools > SysInfo

https://gist.github.com/anonymous/ea455a92be6e773b401d69b65406669d#file-qt_temp_888150484

**Comment by demmm** [comment]
Your log showed you used English keyboard for Live mode and for starting the installer, never switched to another keyboard layout for that.
Then selected fr for the **installed system**, thus typing your password with the english keyboard.  This cannot be marked as a bug as is.

Make sure to either start the Live mode with your desired keyboard layout ('e' during bootup, and set to fr) or in live mode right click the `US` in the panel/systemtray and set to desired layout.
Make sure you see the desired keyboard layout in the panel, before typing a password with language specific characters..

**Comment by Burais** [comment]
I don't know why you want me to boot in live mode, I have installed Kaos on my computer.
My logs say what they want for the first boot on my usb key, that is to say the day I've boot on to install kaos, but now (for 2 month) I'm typing on a french keybord on a kaos installed on my mac drive : it is not a live mode.
I have changed my keybord in kde settings
The fact is : when I boot my machine (almost every day) the french keybord is **not** loaded when I type in the "login window". After, when the KDE desktop is loaded, my keyboard is right and if I disconnect to reconnect, here, the french keyboard is right.
Like I explained in my bug report.
So Yes it is a bug.
